ci_framework/roles/devscripts/tasks/sub_tasks/15_user.yml

```diff
@@ -14,11 +14,10 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-- name: Create the specified user.
-  become: true
-  ansible.builtin.user:
-    name: "{{ cifmw_devscripts_user }}"
-    state: present
+
+- name: Gather the user
+  ansible.builtin.set_fact:
+    cifmw_devscripts_user: "{{ lookup('env', 'USER') }}"
```

Comment:
this `lookup()` will run local on the ansible controller - so it will not match the user on the remote host in case of remote execution. You should use `ansible_user_id` which is always set

Reply:
Thank you. I need to run these remotely. It should be fixed in the latest commit.